dataprofiler/reports/graphs.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+import math
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+
+def plot_histograms(profiler, columns=None):
+    """
+    Take a input of StructuredProfiler class and a list of specified column
+    names and then plots the histograms of those that are int or float
+    columns.
+
+    :param profiler: StructuredProfiler variable
+    :param columns: list of column names to be plotted
+    :type profiler: StructuredProfiler
+    :type columns: list
```

Comment:
You should organize them by parameter, then type of that parameter, then parameter, type of that parameter, and so on... So instead of this the list (top to bottom) should be `:param profiler:, :type profiler:, :param columns:, :type columns:`)